executor/src/witgen/data_structures/finalizable_data.rs

```diff
@@ -215,6 +274,41 @@ impl<T: FieldElement> FinalizableData<T> {
         }
     }
 
+    /// Returns an iterator over the values known in that row together with the PolyIDs.
+    #[auto_enum(Iterator)]
+    pub fn known_values_in_row(
+        &self,
+        row: usize,
+    ) -> impl Iterator<Item = (PolyID, T)> + use<'_, T> {
```

Comment:
```suggestion
    ) -> impl Iterator<Item = (PolyID, T)> + '_ {
```

This compiles! What does `use` mean in this context? 👀

Reply:
I don't know, the compiler suggested it...